### ISSUE TITLE
[Feature Request] Quest Refresh timer

### ISSUE THREAD

**Issue by Kyle-Newton** [Ephenia/Pokeclicker-Scripts#80]
Ability to set a timer for when quests refresh. If afk grinding, it's useful to have quests refresh automatically for things I'm not going to bother with (like some farming tasks).

**Comment by Ephenia** [comment]
You can always filter them out if you don't want to do them with the latest update and quests will auto refresh when you've done all the types of quests that you want completed, see commit 9aef169eaec0c29489521d506ac8bd23916ccedc. It should suffice for what you're essentially wanting, I believe.

**Comment by Kyle-Newton** [comment]
ooo, Thank you. I wasn't aware of this feature and it wasn't very apparent but I've tested it out. I can see the use for this, but it's not quite what I'm after. I want to basically be afk grinding and have it reset anything I haven't done yet, every 10 minutes (or whatever I set). I do like the previous suggestion of a button to press to bring you to a quest location, but that's still more for active playing. 

The issue is if I want to be mostly afk, there's a very limited amount of quests I can do. However if it's refreshing every ten minutes I would be more likely to run into completable quests. Think afk farming some water area for my 10,000 and it asking me to collect fire gems. I don't want to completely filter out gems, because if it rolls collecting water gems, it'll likely complete it in a minute or two. Whereas that fire gem task will roadblock me until I notice and go refresh manually.
